ui/src/reducers/queue.js

```diff
@@ -23,8 +23,8 @@ function queueReducer(state = INITIAL_STATE, action = {}) {
     case 'CLEAR_QUEUE': {
       return {
         ...state,
-        queue: INITIAL_STATE.queue,
-        currentSampleID: INITIAL_STATE.current,
```

Comment:
It should be `INITIAL_STATE.currentSampleID` here, but I think duplicating the actual values is much clearer.